.ci/scripts/create-arn-table.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+set -o pipefail
+
+#
+# Create the AWS ARN table given the below environment variables:
+#
+# - AWS_FOLDER      - that's the location of the publish-layer-version output for each region.
+# - SUFFIX_ARN_FILE - that's the output file to be stored in the AWS_FOLDER.
+# - RELEASE_NOTES_URL - that's the URL for the changelog in the elastic.co
+#
+
+ARN_FILE=${SUFFIX_ARN_FILE}
+
+{
+	echo "For more information, please see the [changelog](${RELEASE_NOTES_URL})."
+	echo ''
+	echo "### ARN"
```

Comment:
Perhaps a more descriptive header:

```suggestion
	echo "### Elastic APM Node.js agent layer ARNs"
```

(Later, it might be helpful to have a link back to the Lambda setup docs from this release notes section.)